lib/linguist/languages.yml

```diff
@@ -1339,6 +1339,7 @@ GDB:
   language_id: 122
 GDScript:
   type: programming
+  color: "#478cbf"
```

Comment:
Yep, I see...
Unfortunately this is the official color 😞 
I think I'll revert this to merge the heuristic rule only.